scripts/utils/helpers.ts

```diff
@@ -10,12 +13,10 @@ function check(flag, message) {
 
 // The current version of the contracts in the repo uses readOracle as the api
 // The old api was peg(), so the currently deployed contracts need manual calling until the upgrade
-async function readOracle(oracleRef, web3) {
-  const data = await web3.eth.call({ to: oracleRef.address, data: web3.eth.abi.encodeFunctionSignature('peg()') });
+export async function readOracle(oracleRef, web3) {
+  const data = await ethers.provider.call({
+    to: oracleRef.address,
+    data: web3.eth.abi.encodeFunctionSignature('peg()')
```

Comment:
seeing web3 and ethers together is starting to mess with me. How hard will it be to axe web3 completely?

Reply:
SOON